site/en/blog/soft-navigations-experiment/index.md

```diff
@@ -17,13 +17,17 @@ tags:
  - origin-trials
 ---
 
+{% Aside 'update' %}
+The Soft Navigations origin trial has entered a [second origin trial from Chrome 117](/origintrials/#/view_trial/21392098230009857) after taking on board feedback from the initial trial.
```

Comment:
"Board" feedback? Did you mean "broad" here?

Reply:
No. Is this not used in US English? https://ludwig.guru/s/take+feedback+on+board

Reply:
I'd go with the expression literally as given in the link there, then: "...after taking **feedback on board** from the initial trial."

Reply:
```suggestion
The Soft Navigations origin trial has entered a [second origin trial from Chrome 117](/origintrials/#/view_trial/21392098230009857) after taking feedback on board from the initial trial.
```